misk/src/main/kotlin/misk/web/PathPattern.kt

```diff
@@ -8,14 +8,24 @@ import java.util.regex.Pattern
  * like this: `{username:[a-z]+}`. If no regex is specified the variable is a sequence of non-'/'
  * characters.
  */
-data class PathPattern(
-        val pattern: Pattern,
+class PathPattern(
```

Comment:
why remove data? because you want to control the equals/hashCode explicitly?

Reply:
correct